From 1a0092db98394c3a55554ff460b48aeade5d54fb Mon Sep 17 00:00:00 2001
From: Hains van den Bosch <hainsvdbosch@ziggo.nl>
Date: Sat, 8 Sep 2018 23:41:32 +0200
Subject: [PATCH] enigma2: adapt network to new ip formatting

Introduced with Busybox 1.29.2

OLd (1.27.2).:
2: eth0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc pfifo_fast qlen 1000\    link/ether 00:09:34:1c:3f:a8 brd ff:ff:ff:ff:ff:ff
2: eth0    inet 192.168.178.185/24 brd 192.168.178.255 scope global eth0
2: eth0    inet6 fe80::209:34ff:fe1c:3fa8/64 scope link \       valid_lft forever preferred_lft forever

New (1.29.2):
2: eth0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc pfifo_fast qlen 1000
    link/ether 00:09:34:1c:3f:a8 brd ff:ff:ff:ff:ff:ff
    inet 192.168.178.185/24 brd 192.168.178.255 scope global eth0
    inet6 fe80::209:34ff:fe1c:3fa8/64 scope link
    valid_lft forever preferred_lft forever
---
 lib/python/Components/Network.py | 26 +++++++++++++-------------
 1 file changed, 13 insertions(+), 13 deletions(-)

diff --git a/lib/python/Components/Network.py b/lib/python/Components/Network.py
index 56c145828..9ec3f0aa6 100644
--- a/lib/python/Components/Network.py
+++ b/lib/python/Components/Network.py
@@ -71,7 +71,7 @@ class Network:
 		ipRegexp = "[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}"
                nameserverPattern = re.compile("nameserver +" + ipRegexp)
+		ipLinePattern = re.compile(ipRegexp + '/')
 		ipPattern = re.compile(ipRegexp)
-- 


